src/flint/fmpz.jl

```diff
@@ -1344,7 +1344,7 @@ moebius_mu(x::Integer) = moebius_mu(fmpz(x))
 > $y \leq 0$, we throw a `DomainError()`.
 """
 function jacobi_symbol(x::fmpz, y::fmpz)
-   y <= 0 && throw(DomainError(y, "Modulus must be positive"))
+   (y <= 0 || mod(y, 2) == 0) && throw(DomainError(y, "Modulus must be odd and positive"))
```

Comment:
:+1: I was thinking the same while reviewing the first commit!

Reply:
It was left out original for performance reasons. The mod is expensive here, and an iseven would be better. But it's not implemented. Another one for someone with some Time TM.

Reply:
Ah right, I just made a PR for that after seing this `mod` here.